File: Assignments/Assignment 35/Java Codes/program35_5.java
// Write a program which accepts a string from user and display it in reverse order.

import java.util.*;

class Test 
{
    void Difference(String str)
    {
        int iCnt  = 0;
        char Arr[] = str.toCharArray();
        int Counter = Arr.length;

        Counter = Counter - 1;

        for(iCnt = Counter; iCnt >= 0; iCnt--)       
        {
            System.out.printf("%c", Arr[iCnt]);
        }
    }
}

class program35_5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter String : ");
        String Arr = sobj.nextLine();

        Test tobj = new Test();

        tobj.Difference(Arr);
    }
}

File: Assignments/Assignment 35/C Codes/program35_5.c
// Write a program which accepts a string from user and display it in reverse order.

#include<stdio.h>

void Reverse(char str[])
{
    char *temp = str;

    while(*str != '\0')
    {
        str++;
    }

    str--;

    while(str >= temp)
    {
        printf("%c", *str);
        str--;
    }
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter String : \n");
    scanf("%[^'\n']s", Arr);

    Reverse(Arr);

    return 0;
}